source/_authors/crell.twig

```diff
@@ -0,0 +1,9 @@
+---
+title: Larry Garfield
+identifier: crell
+twitter: crell
+avatar: # TODO - REPLACE
```

Comment:
How do you want me to get you a pic?

Reply:
I think any link that you consider "stable" is good enough. Gravatar would be good I guess.